### PR TITLE
feat(events-v2) Rough in the events chart on events v2

### PR DESCRIPTION
**Pull request by markstory**
The API to fetch events by user count hasn't landed yet but this gets the ball rolling. I've adjusted the layout to match what it is the mockups as well.

Screen Shot 2019-05-24 at 3 49 18 PM

Refs SEN-662

### CODE REVIEW

**Comment by lynnagara** [comment]
Was it not landed as part of https://github.com/getsentry/sentry/commit/8e5a64f10cdf6a47ae4af25cf278abbd4b5c60fa?

File: src/sentry/static/sentry/app/views/organizationEventsV2/events.jsx [hunk 1]
```diff
@@ -15,10 +18,15 @@ import {getQuery} from './utils';
 
 class Events extends AsyncComponent {
   static propTypes = {
+    router: PropTypes.object,
     organization: SentryTypes.Organization.isRequired,
     view: SentryTypes.EventView.isRequired,
   };
 
+  state = {
+    zoomed: false,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/events.jsx]: Is this only to force a re-render?

**Review by billyvg — APPROVED** [review]

**Comment by markstory** [comment]
> Was it not landed as part of 8e5a64f?

I didn't see that had merged when I started this, and was heads down for a bit. I'll rework this to use the new endpoint data.

File: src/sentry/static/sentry/app/views/organizationEventsV2/events.jsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationEventsV2/events.jsx]: I was replicating the behavior in the current iteration of the events view. It also has a `handleZoom` method that sets `zoomed` to true implying the default should be false.

**Comment by markstory** [comment]
Updated with the y-axis switcher;

File: src/sentry/static/sentry/app/views/organizationEventsV2/events.jsx [hunk 2]
```diff
@@ -43,17 +56,29 @@ class Events extends AsyncComponent {
...
     const {data, dataPageLinks, loading} = this.state;
     const query = location.query.query || '';
 
     return (
-      <div>
+      <React.Fragment>
+        <Panel>
+          <EventsChart
+            router={router}
+            query={location.query.query}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/events.jsx]: there's a local var `query`

**Review by billyvg — APPROVED** [review]

**Pull request merged**